tests/src/MessageLayout.cpp

```diff
@@ -0,0 +1,123 @@
+#include "Application.hpp"
+#include "common/Literals.hpp"
+#include "controllers/accounts/AccountController.hpp"
+#include "controllers/commands/Command.hpp"
+#include "controllers/commands/CommandController.hpp"
+#include "controllers/hotkeys/HotkeyController.hpp"
+#include "messages/layouts/MessageLayout.hpp"
+#include "messages/MessageBuilder.hpp"
+#include "messages/MessageElement.hpp"
+#include "mocks/EmptyApplication.hpp"
+#include "singletons/Emotes.hpp"
+#include "singletons/Fonts.hpp"
+#include "singletons/Paths.hpp"
+#include "singletons/Settings.hpp"
+#include "singletons/Theme.hpp"
+#include "singletons/WindowManager.hpp"
+#include "widgets/Notebook.hpp"
+#include "widgets/splits/Split.hpp"
+#include "widgets/splits/SplitInput.hpp"
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <QDebug>
+#include <QString>
+
+#include <memory>
```

Comment:
Commenting to remember to remove unused headers

Reply:
I've gone ahead and done this in [`269e77d` (#5243)](https://github.com/Chatterino/chatterino2/pull/5243/commits/269e77d304edfbe444fc9fa73d01c2718330facd)